### ISSUE TITLE
Hidden and linked files not showing

### ISSUE THREAD

**Issue by pitylee** [h3imdall/ftp-remote-edit#333]
In the last release you removed the config "Show hihdden files", and in the same time, the linked files won't show in tree view.

Can you patch it, or help me fix it please?

**Comment by cschindl** [comment]
Hi @NoExtraSauce,

to help you, I'd need to know what kind of FTP server is running there exactly? Please take a screenshot of the FTP log. This is under the structure tree and can be extended.

![63637783-32fc7f00-c681-11e9-8065-d439252d9a7c](https://user-images.githubusercontent.com/28109094/64127731-3eebed80-cdb3-11e9-8237-df2946f09863.png)

**Comment by pitylee** [comment]
I can not open this panel, and the small green light is not there either.
Yet I can tell you I am using ssh with key.

**Comment by cschindl** [comment]
Hi @pitylee,

the "Show hidden files" option was only used for FTP connections. Because you could force the FTP server to show hidden files with an additional parameter. SFTP (SSH) does not use this option and has never used it. The SSH library hasn't been changed, so your error can't be caused by the plugin. Could it be that settings or rights on your server have changed so that you can no longer access certain files and they are therefore no longer displayed?

**Comment by pitylee** [comment]
No, the files and folders are there, they are symlinks, and I can see in np++.
As long as I know it is not a server config concern, yet I know it is not the same as it is with the ftp servers show hidden files setting.
I am not aware of plugins that I might have that are in conflict, and would cause such a strange behaviour.

**Comment by cschindl** [comment]
This is also strange because symlinks have never been supported before. When listing a directory, it is not possible to determine if symlinks point to directories or files. However, this is necessary to be able to display them clearly in the tree view. Are you sure the missing files have always been symlinks?

**Comment by pitylee** [comment]
Of course, but I never saw them.

**Comment by cschindl** [comment]
So it has nothing to do with the last release. So is it generally about symlinks not being displayed? If so, I change the issue to a feature request.

**Comment by dkoppstein** [comment]
I support this feature request!
